src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -1791,6 +1833,43 @@ let addBuildingBlockAt (excelIndex: int) (newBB: CompositeColumn) (table: Table)
         |> List.iteri (fun ci header -> ExcelHelper.addColumnAndRows (float (excelIndex + ci)) table header bodyValues.[ci] |> ignore)
     }
 
+type BodyCellType =
```

Comment:
This type will be added to Shared as CompositeCellDiscriminate in my pr #549

Reply:
keep it here and add a [<System.Obsolete("Replace with Shared CompositeCellDiscriminate")>]

Reply:
fixed